### ISSUE TITLE
Motion-blur not supported

### ISSUE THREAD

**Issue by MrLixm** [MrLixm/KUI#5]
Add support of multiple time samples for points.

It seems logical to apply this modification only on `commons` transformations attributes and ignore `arbitrary` and `sources`.

See RenderMan instance array example :

```lua
local numTimeSamples = pointAttr:getNumberOfTimeSamples()
local matrixArrayMap = {}

-- to get motion blur on the instances, create an instanceMatrix at each
-- time sample available from the point cloud points attribute
for idx=0, numTimeSamples-1 do
    
    local sampleTime = pointAttr:getSampleTime(idx)
    local points = pointAttr:getNearestSample(sampleTime)

    -- Each instance in array has its own matrix
    local matrixArray = {}
    local workMatrix = Imath.M44d():toTable()
    -- [...]
    matrixArray[#matrixArray+1]=workMatrix[j]
    -- [...]
    -- set the instanceMatrix for this time sample
    matrixArrayMap[sampleTime] = matrixArray

end

```

**Comment by MrLixm** [comment]
Creating motion-blur tables could have a memory/perf costs.
Would be smart to include an user option to disable/enable the option

**Comment by MrLixm** [comment]
This is now fixed in 2.0.0.
https://github.com/MrLixm/KUI/releases/tag/v2.0.0
